guide/06-imagery-and-raster-analysis/creating_imagery_layers.ipynb

```diff
@@ -0,0 +1,474 @@
+{
```

Comment:
<p>By using the <a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.analytics.html#copy-raster" target="_blank">copy_raster</a> <strong>function</strong><em> </em>you can create one imagery layer by mosaicking multiple images.</p><p> </p><ul><li>Using the&nbsp;<code>raster_type_name</code>&nbsp;parameter, you can specify the raster type, which identifies metadata such as georeferencing, acquisition date, sensor type, and band wavelengths, along with a raster format. The supported raster types can be found in the&nbsp;<a href="https://app.reviewnb.com/Esri/arcgis-python-api/pull/1036/#Supported-raster-types" target="_blank">table</a> at the end of this guide. </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>